recipes/at-cascade/recipe.yaml

```
@@ -0,0 +1,77 @@
+schema_version: 1
+
+context:
+  # name of this package
+  name: at-cascade
+  # latest release for this year's stable version of at-cascade
+  version: "2025.0.3"
+  # Minimum python version that has not reached end of life.
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/bradbell/at_cascade/archive/${{ version }}.tar.gz
+  sha256: f2c70fa39464f825134da09ab14fc671ffb5c74218809433ff962f92151aed6e
+  # patches:
+  #   - ${{ version }}.patch
+
+# build.sh and build.bat run all the at_cascade tests
+# using the code in its source directory (sandbox).
+#
+# They then install at_cascade using pip as in commented out scipt entry below.
+build:
+  number: 0
+  noarch: python
+  # script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python >=${{ python_min }}
+    - hatchling
+    - pip
+    - dismod-at >=2025.0.0
+    - numpy
+    - scipy
+  run:
+    - python >=${{ python_min }}
+    - dismod-at >=2025.0.0
+    - numpy
+    - scipy
+
+tests:
+  - python:
+      imports:
+        - at_cascade
+      pip_check: true
+      python_version: ${{ python_min }}.*
+  #
+  # Run one of the sandbox tests using the installed version of at_cascade.
+  - script:
+      # python example/csv/sim_fit_pred.py
```

```suggestion
  - script:
```
These comments are superfluous.